scripts/mutect2_wdl/hapmap_sensitivity_truth.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should this be a supported workflow?  If not, the file location should change to an ``unsupported`` directory.